matrix-meetings-bot/test/MeetingService.test.ts

```diff
@@ -1804,6 +1804,52 @@ describe('test relevant functionality of MeetingService', () => {
         callInfo(0, 3, StateEventName.IO_ELEMENT_WIDGETS_LAYOUT_EVENT)
       ).toStrictEqual(expected);
     });
+
+    test('update room layout when widgets change', async () => {
+      const event = createEvent(parentId);
+      let widgets = [WidgetType.COCKPIT, WidgetType.BREAKOUT_SESSIONS, 'poll'];
+      event.widget_ids = widgets;
+      await meetingService.createMeeting(userContext, event);
+
+      // should not have custom layout
+      verify(
+        clientMock.sendStateEvent(
+          parentId,
+          StateEventName.IO_ELEMENT_WIDGETS_LAYOUT_EVENT,
+          anything(),
+          anything()
+        )
+      ).times(0);
+
+      // add jitsi widget
+      widgets = ['jitsi'];
+      await meetingService.handleWidgets(
+        userContext,
+        new MeetingWidgetsHandleDto(parentId, true, widgets)
+      );
```

Comment:
`handleWIdgets` will load layout only for `'jitsi'` because `'poll'` is not mocked to be a widget in the room